backstop/tests/table.html

```diff
@@ -2412,6 +2412,7 @@ <h2>Additional options</h2>
           </div>
 
           <div class="iui-right">
+            <span class="iui-text-muted">Rows per page</span>
```

Comment:
Shouldn't these also include `iui-text-block`?

Reply:
Applying that class doesn't appear to change anything visually.